client/hospital.lua

```diff
@@ -106,10 +106,10 @@ local function checkInControls(variable)
         if IsControlJustPressed(0, 38) then
             exports['qb-core']:KeyPressed(38)
             if variable == "checkin" then
-                TriggerEvent('qb-ambulancejob:checkin')
+                checkIn()
                 listen = false
             elseif variable == "beds" then
-                TriggerEvent('qb-ambulancejob:beds')
+                putPlayerInClosestBed()
                 listen = false
             end
```

Comment:
Non obligatory, but could be a decent future change. Turning this into a switch statement could improve appearance.

Reply:
Maybe. I'm still not sure if I believe in a map of functions to replicate a switch statement.